docs/presentation/Setup/index.md

```diff
@@ -1,57 +1,65 @@
 # Setting Up iTwin.js Presentation Library
 
-In addition to setting up *imodeljs-core* there are some steps that API
-consumers must do before the library can be used.
+Before the Presentation library can be used, it needs to be properly initialized. Because of the nature of [iTwin.js architecture](../../SoftwareArchitecture.md), an iTwin.js application always has a backend and may or may not have a frontend, which means the library needs to be initialized on the backend and on the frontend if it is used - just like the iTwin.js framework itself.

```

Comment:
```suggestion
Before the Presentation library can be used, it needs to be properly initialized. Because of the nature of [iTwin.js architecture](../../SoftwareArchitecture.md), the library needs to be initialized on the backend and each frontend that is in use — just like the iTwin.js framework itself.
```